common/dynamicconfig/filter.go

```diff
@@ -57,6 +57,8 @@ func ParseFilter(filterName string) Filter {
 		return WorkflowType
 	case "ratelimitKey":
 		return RatelimitKey
+	case "operation":
+		return Operation
```

Comment:
I can't remember if these three locations are all that are necessary 😢 
Are you sure it's complete here, or should I go hunt my past PRs for the places I missed, to double check?

Reply:
let me leave this on PR and try it in 'anger' and figure that out. If it doesn't work I may come back to your offer